src/h5web/toolbar/controls/ScaleOption.module.css

```diff
@@ -0,0 +1,3 @@
+.icon {
+  composes: icon from '../Toolbar.module.css';
+}
```

Comment:
You could keep this in `ScaleSelector.module.css` and import that file in `ScaleOption.tsx`